imap_processing/codice/codice_l1a.py

```diff
@@ -547,12 +547,33 @@ def create_hskp_dataset(
         attrs=cdf_attrs.get_global_attributes("imap_codice_l1a_hskp"),
     )
 
+    # These variables don't need to carry over from L0 to L1a
+    exclude_variables = [
+        "spare_1",
+        "spare_2",
+        "spare_3",
+        "spare_4",
+        "spare_5",
+        "spare_6",
+        "spare_62",
+        "spare_68",
+        "version",
+        "type",
+        "sec_hdr_flg",
+        "pkt_apid",
+        "seq_flgs",
+        "src_seq_ctr",
+        "pkt_len",
+        "chksum",
+    ]
+
     for variable in packet:
-        attrs = cdf_attrs.get_variable_attributes(variable)
+        if variable not in exclude_variables:
+            attrs = cdf_attrs.get_variable_attributes(variable)
```

Comment:
you could this this and undo below line changes.
```suggestion
        if variable in exclude_variables:
             continue

        attrs = cdf_attrs.get_variable_attributes(variable)
```